app/models/cdo/create/country.js

```diff
@@ -0,0 +1,26 @@
+const { routes } = require('../../../constants/cdo/owner')
```

Comment:
Don;t think this file is needed anymore

Reply:
Yeah, this file was deleted on a subsequent commit